frontend/src/Components/Elements/Table/TableModal.styled.js

```diff
@@ -82,7 +82,9 @@ export const CloseButton = styled(FJIcon)`
   }
 `;
 
-export const Download = styled.div``;
+export const Download = styled.div`
```

Comment:
This might be a pre-existing style, but I noticed the Download Data button has a blue underline for the text, which looks a bit odd:

<img width="251" alt="image" src="https://user-images.githubusercontent.com/129679/165541577-893bc79c-fa31-4ad2-bb9d-66373e7119d5.png">